src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java

```diff
@@ -87,9 +86,9 @@ public void testWatchWithNoKeyAffection(Jedis jedis) throws ExecutionException,
         runAsync(() -> anotherJedis.set(ANOTHER_KEY, ANOTHER_VALUE)).get();
         transaction.set(FIRST_KEY, SECOND_VALUE);
         List<Object> result = transaction.exec();
-        assertEquals(1, result.size());
-        assertEquals("OK", result.get(0));
-        assertEquals(SECOND_VALUE, jedis.get(FIRST_KEY));
+        assertThat(result).hasSize(1)
```

Comment:
`containsExactly` is sufficient here, no need to check for size

Reply:
Fixed